tests/Teams/ActionBasedMessagingExtension-FetchTask/AdaptiveCardHelper.cs

```diff
@@ -0,0 +1,108 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using AdaptiveCards;
+using Microsoft.Bot.Schema.Teams;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.BotBuilderSamples
+{
+    public static class AdaptiveCardHelper
+    {
+        public static SubmitExampleData GetSubmitExampleDataFromAction(MessagingExtensionAction action)
```

Comment:
Extension method on MessageExtensionAction? (just an idea to explore?)

Reply:
Changed to ToSubmitExampleData